src/conventional/sonde_tac2ioda.py

```diff
@@ -961,6 +965,11 @@ def change_vars(profile):
 
     # SPECIAL:  Most soundings launched 50-55 minutes prior to stated synoptic time, so a 12Z
     # launch is usually initiated close to 11:05Z.
+    target_hr = target_date[11:13]
+    if target_hr == '12' and profile['hour'] == 11:
```

Comment:
are you not worried about the (albeit fractionally sized) cases where balloons are launched at 06z or 18z?

Reply:
What about converting target_hr to a datetime object and using timedelta, if it is -1 hour, then set profile['hour'], I think this would result in no need for an `elif`

Reply:
Agreed, I am worried about 6/18z a little. I pondered it, just not sure how reliable it will end up.  Going 100% to BUFR would totally help, but that is a wish-cast.

Meanwhile, I also thought about the datetime object in place of what I did here.  I will give it a go with some 00z data because I fear the TTAA YYGGi WMO_number stuff will have YY (day-of-month) set by synoptic time and GG which should be release time (23) misaligned.  I was surprised to read the WMO manual on codes says to put launch hour in the GG section and USA and other countries are setting that to synoptic hour, not actual launch hour.  Arghhh.  I'll check a bunch of worldwide 00z soundings and see what their pattern is.  Not following standards globally stinks.

Reply:
we can just flat out reject the outliers for now, or specify a data quality flag

Reply:
@CoryMartin-NOAA  It turns out that I must do a stricter check on date/time because some reports centered at 00z contain the new day and 00 for hour, while others have the prior day and 23z (even 22z in some) which would then assign a completely wrong date.  I have the fix being tested now.

Reply:
Fixed.